doc/config.example.yaml

```
@@ -85,3 +85,14 @@ dns:
 logging:
   # Values: Trace, Info, Warn, Error, Fatal, None (default)
   level: None
+
+################################################################################
+##                               Event Handlers                               ##
+################################################################################
+event_handlers:
+  # URLs to push a data when a block is externalized. (path is "/push")
+  block_externalized:
+    - http://127.0.0.1:3836
+  # URLs to push a data when a pre-image is updated. (path is "/push_preimage")
+  preimage_received:
+    - http://127.0.0.1:3836
```

Let's make them match the event name for simplicity.
So:
```suggestion
  # URLs to push a data when a block is externalized. (path is "/block_externalized")
  block_externalized:
    - http://127.0.0.1:3836
  # URLs to push a data when a pre-image is updated. (path is "/preimage_received")
  preimage_received:
    - http://127.0.0.1:3836
```

---

@Geod24 
First of all, thank you for your review.
But I had a better idea, so I tested a few things.
Please check this. 
I think we can unify the handlers in this way.
https://gist.github.com/MichaelKim20/2ee0fd4590cc0a9d56adf5aa0c94b020